README.md

```diff
@@ -103,12 +103,12 @@ for more details.
       * `mpaso.rst.0002-01-01_00000.nc` (or any other mpas-o restart file)
       * `streams.ocean`
       * `mpas-o_in`
-  * mpas-cice files:
-      * `mpascice.hist.am.timeSeriesStatsMonthly.*.nc`
-      * `mpascice.rst.0002-01-01_00000.nc` (or any other mpas-cice restart
+  * mpas-seaice files:
+      * `mpasseaice.hist.am.timeSeriesStatsMonthly.*.nc`
+      * `mpasseaice.rst.0002-01-01_00000.nc` (or any other mpas-seaice restart
```

Comment:
For the time being, I would keep the old filenames in there as well. We can remove them all together when the change to MPAS-seaice is definite.